apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -441,6 +441,61 @@ export class ModalPage {
     await allWalletsListSearchItem.click()
   }
 
+  async clickTabWebApp() {
+    const tabWebApp = this.page.getByTestId('tab-webapp')
+    await expect(tabWebApp).toBeVisible()
+    await tabWebApp.click()
+  }
+
+  async clickCopyLink() {
+    const copyLink = this.page.getByTestId('wui-link-copy')
+    await expect(copyLink).toBeVisible()
+
+    let hasCopied = false
+
+    while (!hasCopied) {
+      await copyLink.click()
+      await this.page.waitForTimeout(500)
+
+      const snackbarMessage = this.page.getByTestId('wui-snackbar-message')
+      const snackbarMessageText = await snackbarMessage.textContent()
+
+      if (snackbarMessageText && snackbarMessageText.startsWith('Link copied')) {
+        hasCopied = true
+      }
+    }
+
+    return this.page.evaluate(() => navigator.clipboard.readText())
+  }
```

Comment:
I think there was an issue with shared clipboard on test runners cc @chris13524 @lukaisailovic

Reply:
Mhmm yeah i had issues, but once i added [this](https://github.com/reown-com/appkit/blob/f5c8ddf27b60cb7599372ef4eb7f029f9fd6b6cb/apps/laboratory/tests/wallet-features.spec.ts#L24-L28) it got fixed.